src/events/event-attendance.controller.ts

```diff
@@ -125,6 +125,20 @@ export class EventsAttendanceController {
   async create(
     @Body() { id, ...data }: EventAttendanceCreateDto,
   ): Promise<EventAttendanceDto> {
+    const checkVisitor = await this.repository
```

Comment:
This query gets all entries where this contact was a visitor, does not matter for which group.
I would expect `'"contactId"=:contactId AND "groupId"=:groupId', ` so that if there are 0 entries, then the person has never attended this group's events

Reply:
I have added that logic to the code. Members are only flagged as guests when it is their first time in a particular group.

![image](https://user-images.githubusercontent.com/5897163/117458310-73318d00-af4a-11eb-8c4c-75381195e6df.png)